src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,14 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * By default return false;
```

Comment:
Selecting this kinda default behavior is not suggested. While in this case it works, it shouldn't be the norm.
Think why?

BTW, you need to tell what this method does. Return false is hardly informative!